### ISSUE TITLE
TAXII Server error

### ISSUE THREAD

**Issue by yarbs** [mitre/cti#104]
Changes within the last several months have left the taxii server , https://cti-taxii.mitre.org/taxii/, inoperable.  Basic calls to the server result in the following error, "requests.exceptions.HTTPError: 406 Client Error: Not Acceptable for url: https://cti-taxii.mitre.org/taxii/".

After tracing this error, it appears the taxii server believes there to be an invalid content type of 'application/taxii+json'.

I apologize in advance if this is not the correct place to report this issue.

<img width="1354" alt="mitre_taxii_01_20200820" src="https://user-images.githubusercontent.com/2685379/90771739-591b2600-e2b9-11ea-8fbe-34b8f1e7e6d2.png">
<img width="1211" alt="mitre_taxii_02_20200820" src="https://user-images.githubusercontent.com/2685379/90771746-5c161680-e2b9-11ea-8d2a-5c6f498e5363.png">

**Comment by isaisabel** [comment]
Hi @yarbs,

We're currently in the process of improving our USAGE documentation to address this issue (#103). Essentially the taxii2client library changed their API and therefore depending on your installed version you may be trying to access using TAXII 2.1 instead of TAXII 2.0 (our server only supports the latter). See [the WIP taxii2client import instructions](https://github.com/mitre/cti/blob/docs/usage-imports/USAGE.md#taxii2client) for guidance on how to fix this issue. 

**TLDR:** update your import of `taxii2client` to use `v20`, e.g:
```
from taxii2client.v20 import Collection
```

**Comment by yarbs** [comment]
Outstanding!  I swear I tried this before but I appreciate the point in the right direction.
